src/App.tsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
+import { useState } from 'react';
+
 import 'bulma/css/bulma.css';
+import cn from 'classnames';
+
 import './App.scss';
+import { GoodsList } from './Components/GoodsList/GoodsList';
```

Comment:
improve react imports order

https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf